### DISCUSSION TITLE
Is it possible to create a local component in page dir in Nuxt 3?

### DISCUSSION THREAD

**Discussion by pheng0077** [nuxt/nuxt#15912] [Questions] [answered]
I have a very complex page, but if I put them all in one page file, it becomes messy and hard to manage. So, is it possible to create a component inside the page directory?

Example
![image](https://user-images.githubusercontent.com/43340923/212839547-bcddfc50-de97-4fb8-8fa1-706c0547b45b.png)

**Comment by TheAlexLichter** [comment, marked as answer]
Not by default. You could probably use the `ignorePrefix` and put it in there but there are better options:

* Use the `components` folder with subfolders
* Use a `layer` for the admin page (via `extends`)

**Reply by pheng0077** [reply]
Thank you so much :)
